go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/proemergotech/chrome-webrtc-packet-log-parser
+
+go 1.14
+
+require (
+	github.com/golang/protobuf v1.4.1
+	github.com/pion/rtcp v1.2.1
+)
+
+replace github.com/pion/rtcp => /home/pongi/gohack/github.com/pion/rtcp
```

Comment:
remove this

Reply:
Can't remove til pion doesn't merge our PR.
Updated to point to github.com/proemergotech/rtcp.